cluster.go

```diff
@@ -259,6 +259,33 @@ func (c *clusterClient) _refresh() (err error) {
 			}
 			if len(g.nodes) > 1 {
 				n := len(g.nodes) - 1
+
+				if c.opt.EnableReplicaAZLookUp {
+					var (
+						mu sync.Mutex
+						wg sync.WaitGroup
+					)
+					for i := 0; i < n; i += 4 { // batch 4 nodes for each goroutine
+						for j := i; j < i+4 && j < n; j++ {
+							replica := g.nodes[j+1]
+							rConn := conns[replica.Addr].conn
+
+							wg.Add(1)
+							go func(j int, rConn conn) {
```

Comment:
```suggestion
							go func(j int, rConn conn, replica *ReplicaInfo) {
```
The `mu` is not necessary if we pass `*ReplicaInfo` into the goroutine.

Reply:
[afae5ee](https://github.com/redis/rueidis/pull/721/commits/afae5ee597ade7dbb2bc51191acee3253f371754)

Oh, I found that event `ReplicaInfo` doesn't need.